improver_tests/wxcode/wxcode/test_utilities.py

```diff
@@ -590,6 +591,12 @@ def modify_tree_fixture(node, key, value):
             ["kittens"],
             "Node lightning has a non-numeric probability threshold ['kittens']",
         ),
+        (
+            "mist_conditions",
+            "if_true",
+            "no_precipitation_cloud",
+            "Unreachable node 'fog_conditions'",
+        ),
```

Comment:
@fionaRust: The unreachable node check is tested in this unit test, as well as the acceptance test. Do you think the acceptance test is unnecessary?

Reply:
I guess so, I'm not sure we should test it in both places, and I keep dipping in and out of looking at this PR and getting confused about what's tested where and why. I'm not too bothered though, so I'm going to approve it now anyway and let you decide if you want to remove the failure acceptance test or not.